services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/hakemus/HakemusServiceITest.kt

```diff
@@ -2691,6 +2741,28 @@ class HakemusServiceITest(
     }
 }
 
+private fun Assert<TormaystarkasteluTulos>.isCorrectTormaystarkasteluTulos():
```

Comment:
You could create the expected object and assert that it equals the actual object.

Reply:
This could also be closer to the call site since it's only used in one method.

Reply:
You're right. It is much more clear to create the expected object and do a `isEqualTo` comparion against it. I have been reading your brilliant use of `Assert` extensions and wanted to try that and got carried away. I think I do the same with using trailing commas everywhere even when it makes the code less readable.